Validation/RecoParticleFlow/interface/NicePlot.h

```diff
@@ -4,7 +4,13 @@
 #include <TH1.h>
 #include <TPad.h>
 
-class Style : public TH1 {};
+class Style : public TH1 {
+private:
+  void AddBinContent(Int_t){};
```

Comment:
How come our `clang-tidy` doesn't change these to
```suggestion
  void AddBinContent(Int_t) override {};
```
?

Reply:
good catch @makortel . We pass `-header-filter='/build/muz/xx/CMSSW_15_0_CLANG_X_2025-01-28-2300/src/.*'` to clang-tidy. Looks like it is not working any more [a] but using `-header-filter='src/.*'` works [b]. I will update the build rules to use relative path

[a]
```
> clang-tidy -export-fixes tmp/el8_amd64_gcc12/code-checks/Validation/RecoParticleFlow/src/NicePlot.cc.yaml -header-filter='/build/muz/xx/CMSSW_15_0_X_2025-01-29-1100/src/.*' src/Validation/RecoParticleFlow/src/NicePlot.cc
375 warnings generated.
Suppressed 375 warnings (375 in non-user code).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
```

[b]
```
> clang-tidy -export-fixes tmp/el8_amd64_gcc12/code-checks/Validation/RecoParticleFlow/src/NicePlot.cc.yaml --header-filter='src/.*'  src/Validation/RecoParticleFlow/src/NicePlot.cc
375 warnings generated.
src/Validation/RecoParticleFlow/interface/NicePlot.h:9:8: warning: annotate this function with 'override' or (rarely) 'final' [modernize-use-override]
    9 |   void AddBinContent(Int_t){};
      |        ^                   
      |                             override
src/Validation/RecoParticleFlow/interface/NicePlot.h:10:8: warning: annotate this function with 'override' or (rarely) 'final' [modernize-use-override]
   10 |   void AddBinContent(Int_t, Double_t){};
      |        ^                             
      |                                       override
src/Validation/RecoParticleFlow/interface/NicePlot.h:11:12: warning: annotate this function with 'override' or (rarely) 'final' [modernize-use-override]
   11 |   Double_t RetrieveBinContent(Int_t) const { return 0.; };
      |            ^
      |                                            override
src/Validation/RecoParticleFlow/interface/NicePlot.h:12:8: warning: annotate this function with 'override' or (rarely) 'final' [modernize-use-override]
   12 |   void UpdateBinContent(Int_t, Double_t){};
      |        ^                                
      |                                          override
```